spec/classes/redis_sentinel_spec.rb

```diff
@@ -52,15 +52,15 @@
       'mode'    => '0644',
       'owner'   => 'redis',
       'content' => $expected_noparams_content
-      )
```

Comment:
This was also generated by another change.

Reply:
Doing fixes one cop at a time, I make sure I don't unfix cops I've already fixed (or introduce entirely new failing cops).  For the ones I haven't, the number of failures can and does tend to increase though.